custom-vu/src/main/kotlin/jces1209/vu/page/customizecolumns/DcColumnsEditor.kt

```diff
@@ -0,0 +1,10 @@
+package jces1209.vu.page.customizecolumns
+
+import org.openqa.selenium.WebDriver
+
+class DcColumnsEditor(
+    private val driver: WebDriver
+) : ColumnsEditor(driver) {
```

Comment:
If there are no differences between Cloud and DC impls, there's no need to create them. [Occam's razor](https://en.wikipedia.org/wiki/Occam%27s_razor): "unnecessary entities should be removed".